examples/pulsar/values-oauth2.yaml

```diff
@@ -0,0 +1,47 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+images:
+  broker:
+    # the image should contain the `io.streamnative.pulsar.broker.authentication.AuthenticationProviderOAuth` lib
+    repository: streamnative/sn-platform
+    tag: 2.9.2.19
+  proxy: # if the proxy component is enabled
+    repository: streamnative/sn-platform
+    tag: 2.9.2.19
+
+auth:
+  authorization:
+    # if set to true, please set one of the superRoles to your super-application-id
+    enabled: false 
+  authentication:
+    enabled: true
+    provider: "oauth2"
+    # below configurations are using Azure as the provider
+    # please refer to: https://streamnative.slab.com/posts/how-oauth-2-broker-plugin-work-with-azure-active-directory-azure-ad-platform-ui9a8ox8
```

Comment:
I think we should remove these private links in this public repo

Reply:
ok, then I will add a doc in this pr to describe how to use azure as the oauth2 provider